python/sglang/srt/server_args.py

```diff
@@ -1261,6 +1261,12 @@ def _handle_hicache(self):
                 )
 
     def _handle_speculative_decoding(self):
+        if (
+            self.speculative_draft_model_path is not None
+            and self.speculative_draft_model_revision is None
+        ):
+            self.speculative_draft_model_revision = "main"
```

Comment:
Why we need to set this by default?

Reply:
@Qiaolin-Yu sorry, lost track of this. 

If we don't set a default value for it, it would default to None and pass in [here](https://github.com/sgl-project/sglang/pull/11893/files#diff-65c6ac2c41977f68e460f18e35053b97089631f88a9958b0796343fccee78a67R528), then [from_server_args](https://github.com/sgl-project/sglang/blob/main/python/sglang/srt/configs/model_config.py#L235) would use server_args.revision as draft model revision, which introduce problem.